docs/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.rst

```diff
@@ -36,10 +262,19 @@ class.
     dataframe
     partitioning/partition
     partitioning/partition_manager
-    omnisci_engine
     df_algebra
     expr
     calcite_algebra
     calcite_builder
     calcite_serializer
     omnisci_worker
+
+Column name mangling
+''''''''''''''''''''
```

Comment:
something's up with formatting, I don't see this text in the rendered page whatsoever:
![image](https://user-images.githubusercontent.com/45396231/144875624-41a3d93a-ebf9-40ca-a55a-33be1080e9ab.png)

Reply:
fixed